protocol/0031-ETHB-ethereum_bridge_spec.md

```diff
@@ -148,3 +148,98 @@ This example connects the network to Ropsten:
 * ERC20 smart contract specific requirements:
   * A valid multisig bundle can be passed to the `remove_asset` function to successfully remove a previously  allow listed token (<a name="0031-COSMICELEVATOR-019" href="#0031-COSMICELEVATOR-019">0031-COSMICELEVATOR-019</a>)
   * An invalid multisig bundle is rejected by the `remove_asset` function (<a name="0031-COSMICELEVATOR-020" href="#0031-COSMICELEVATOR-020">0031-COSMICELEVATOR-020</a>)
+
+
+## ERC20 Bridge Logic to Vega Integration Tests (Vega System Tests)
+1.  `address payable public erc20_asset_pool_address;`
```

Comment:
Again, I am guessing this is to be verified on the ethereum side (would be good to clarify).

Reply:
it is verified on chain, but against what the system tests have for those contracts